### ISSUE TITLE
Build issue

### ISSUE THREAD

**Issue by rebokdev** [MyrikLD/LinusTrinus#45]
I'm building using the readme.md from samples and i get this:

/home/rebok/LinusTrinus/samples/driver_linus_trinus/watchdog.cpp:58:45: error: no matching function for call to ‘vr::IVRWatchdogHost::WatchdogWakeUp(vr::ETrackedDeviceClass)’
   58 |         vr::VRWatchdogHost()->WatchdogWakeUp(vr::TrackedDeviceClass_HMD);

**Comment by GabJustGab** [comment]
Quite late with the answer, but I just got that error as well. Apparently, you need to download the [latest version of the OpenVR](https://github.com/ValveSoftware/openvr/releases?page=1) headers and _not_ the ones specified on the [README.md in the samples folder](https://github.com/MyrikLD/LinusTrinus/blob/master/samples/README.md), it seems to solve the issue (at least in my case)

**Comment by rebokdev** [comment]
now i got

/usr/bin/ld: cannot find -lOPENVR_LIBRARIES-NOTFOUND: No such file or directory

maybe that cause i didn't install openvr
but i don't know how
i have searched for this package in fedora and coprs and i can't find

**Comment by 0something0** [comment]
Yup, its not in Fedora repos - https://pkgs.org/search/?q=openvr

I recommend using a Distrobox (https://github.com/89luca89/distrobox) container with Arch Linux, and use `pacman -S` to install the dependencies and compile within the container. 

SteamVR Home is kinda wonky, everything is blue, but I'm not sure if that has anything to do with this.
